apps/src/music/views/MusicView.jsx

```diff
@@ -219,6 +220,7 @@ class UnconnectedMusicView extends React.Component {
   }
 
   async onLevelLoad(levelData, initialSources) {
+    installFunctionBlocks(this.getBlockMode());
```

Comment:
It might be worth doing this in `MusicLabView` so we can use a `useEffect` hook and only install function blocks when the `blockMode` actually changes.

Reply:
done!